src/backend/wine/manager/downloader/main.ts

```diff
@@ -199,7 +199,9 @@ async function installVersion({
   const tarFile =
     installDir + '/' + versionInfo.download.split('/').slice(-1)[0]
   const installSubDir =
-    versionInfo.installDir ?? installDir + '/' + versionInfo.version
+    versionInfo.installDir !== undefined && versionInfo.installDir !== ''
+      ? versionInfo.installDir
+      : installDir + '/' + versionInfo.version
```

Comment:
The only way this change would be different is if `versionInfo.installDir` would be `''`, but in that case this could be simplified to `versionInfo?.installDir ? versionInfo.installDir : installDir + '/' + versionInfo.version`. This would convert a nullish coalescence into a normal falsy check. 

<img width="468" height="223" alt="image" src="https://github.com/user-attachments/assets/309f93aa-5bea-4141-aee4-bc02ac7d8dcd" />